posthog/migrations/0187_event_properties.py

```diff
@@ -0,0 +1,76 @@
+# Generated by Django 3.2.5 on 2021-12-03 09:12
+
+import django.contrib.postgres.indexes
+import django.db.models.deletion
+import django.utils.timezone
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0186_insight_refresh_attempt"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="EventProperty",
+            fields=[
+                ("id", models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
+                ("event", models.CharField(max_length=400)),
+                ("property", models.CharField(max_length=400)),
+                (
+                    "property_type",
+                    models.CharField(
+                        choices=[
+                            ("NUMBER", "Number"),
+                            ("STRING", "String"),
+                            ("BOOLEAN", "Boolean"),
+                            ("DATETIME", "DateTime"),
+                        ],
+                        default="STRING",
+                        max_length=20,
+                        null=True,
+                    ),
+                ),
+                ("property_type_format", models.CharField(max_length=100, null=True)),
```

Comment:
should we constrain the allowed formats too? Each format will need custom query code...